src/modules/CreateClassForm/components/CreateClassForm.js

```diff
@@ -1,23 +1,44 @@
-import React from 'react';
 import PropTypes from 'prop-types';
-import styled from 'styled-components';
-import { px2em } from 'styles/functions';
+import React from 'react';
 import { Form } from 'react-controlled-form';
+import styled from 'styled-components';
 import { Button, FormField, Label, Select, SubmitButton } from 'modules/common';
+import { px2em } from 'styles/functions';
+
+const toOption = ({ id, name }) => ({ label: name, value: id });
 
 const FormGroup = styled.div`margin-bottom: ${px2em(10)};`;
 
-function CreateClassForm({ onSubmit }) {
+const YesNoGroup = ({ label, name, placeholder }) => (
```

Comment:
`placeholder` and `name` never used? Looks like only `id` and `label` are supplied in the JSX.